shinysdr/plugins/wspr/telemetry.py

```diff
@@ -107,11 +108,14 @@ def get_txpower(self):
 
     @exported_value(type=Track, changes='explicit', label='Track')
     def get_track(self):
-        latitude, longitude = grid_to_lat_long(self.__grid)
-        track = Track(
-            latitude=TelemetryItem(latitude, time.time()),
```

Comment:
I fixed: Using time.time() here is incorrect in two ways:

* Because this getter is `changes='explicit'`, the value returned here MUST NOT change except when `self.state_changed()` is also called. (If this were genuinely supposed to be the current time, it should be declared `changes='continuous'`.)
* The semantically correct time value is when the information was received, i.e. `self.__last_heard`.